csrc/scheduler/normalization_inner_outer_tma_ws.cpp

```diff
@@ -791,21 +789,31 @@ void scheduleFusion(Fusion* fusion, const ReductionParams* rparams) {
           ? tma_inline_pos + 1
           : tma_inline_pos;
       for (auto cached_tv : cached_inputs) {
-        if (!cached_tv->hasBroadcast() ||
-            !is_redu_mapped_to_bcast(inner_reference_tv, cached_tv)) {
+        // skip smem tvs as they are TMA loaded and already special inlined
+        if (cached_tv->getMemoryType() == MemoryType::Shared) {
           continue;
         }
-        if (can_vectorize(ir_utils::getSoleProducerTv(cached_tv))) {
-          cached_tv->axis(last_iter_dim)->parallelize(ParallelType::Vectorize);
-        } else {
-          cached_tv->axis(last_iter_dim)->parallelize(ParallelType::Unroll);
+        // skip tvs that are already vectorized
+        if (cached_tv->axis(-1)->getParallelType() == ParallelType::Vectorize) {
```

Comment:
QQ: is it intentional to check `axis(-1)`? Since our schedule below is doing `cached_tv->axis(last_iter_dim)->parallelize(ParallelType::Vectorize);`

Reply:
yes, extended comments to:
```
        // skip tvs that are already vectorized in general vectorization
        // analysis and propagation.
```